File: Selenium/Activities/Activity3.java
package demos;

import org.openqa.selenium.By;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.firefox.FirefoxDriver;

public class Activity3 {
	
	public static void main(String[]args) {
		
		WebDriver driver = new FirefoxDriver();
		driver.get("https://v1.training-support.net/selenium/login-form");
		driver.getTitle();
		System.out.println("Home page title: " + driver.getTitle());
		
		driver.findElement(By.xpath("//input[@id='username']")).sendKeys("admin");
		System.out.println("Username entered");
		
		driver.findElement(By.xpath("//input[@id='password']")).sendKeys("password");
		System.out.println("Password entered");
		
		driver.findElement(By.xpath("//button[@type='submit']")).click();
		
		System.out.println("Logged in successfully");
		
		driver.close();
		
	}

}

File: Selenium/Activities/activity3.py
from selenium import webdriver
from selenium.webdriver.firefox.service import service as FirefoxService
from selenium.webdriver.common.by import By
import time

with webdriver.Firefox() as driver:
    #Open URL
    driver.get("https://v1.training-support.net/selenium/login-form")
    print("Page Opened: ", driver.title)

    driver.find_element(By.XPATH, "//input[@id='username']").send_keys("admin")

    driver.find_element(By.XPATH, "//input[@id='password']").send_keys("password")
    time.sleep(2)
    driver.find_element(By.XPATH, "//button[@type='submit']").click()
    time.sleep(2)

    message = driver.find_element(By.XPATH, "//div[@id='action-confirmation']")
    print("Login message: ", message.text)
